src/containers/Login/Login.vue

```diff
@@ -63,7 +64,6 @@ export default {
         this.$store.commit('customerInformation', dude);
```

Comment:
do you think dude is a serious variable name?

Reply:
why not ?